server/configuration.go

```diff
@@ -50,6 +50,9 @@ func (is *ICEServers) UnmarshalJSON(data []byte) error {
 	if err != nil {
 		return err
 	}
+	if unquoted == "" {
+		return nil
+	}
```

Comment:
This fixes an issue when no STUN servers are passed. Instead of an empty slice of strings we were getting a slice with a single empty string which was making connection fail.